create_framework/http_kernel_httpkernel_class.rst

```diff
@@ -69,7 +69,9 @@ Our code is now much more concise and surprisingly more robust and more
 powerful than ever. For instance, use the built-in ``ExceptionListener`` to
 make your error management configurable::
 
-    $errorHandler = function (HttpKernel\Exception\FlattenException $exception) {
+    use Symfony\Component\Debug\Exception\FlattenException;
```

Comment:
Please add use statement for `ExceptionListener`.
The code example is inconsistent now since there's a use statement for `FlattenException` but not for `ExceptionListener` (line 79).